src/components/network-chart/index.css

```diff
@@ -19,86 +19,82 @@
 }
 
 .networkChartRowItem {
+  /* This color is overridden for specific mime types, see below. */
+  --marker-color: var(--grey-70);
+
   display: block;
   width: 100%;
   height: 16px;
 }
 
+.networkChartRowItem.odd {
+  background-color: #f5f5f5;
+}
+
+.networkChartRowItem:hover {
+  background-color: #bbe0f6;
+}
+
```

Comment:
This was just moved from below. I thought it was easier to read if it was close to the "main" ruleset. I did it for other things below too.